### DISCUSSION TITLE
Dual tarrif for the geofence location

### DISCUSSION THREAD

**Discussion by sbkbg** [teslamate-org/teslamate#3038] [Ideas]
Hi, is there a possibility to add a dual tarrif price (and the beggining and ending time for it) option for the geofence locantions  ?

**Comment by Supermene** [comment]
I was just about to add this idea, in order to have in geofences the option to have a "peak" and "off-peak" price with also the option to set the timetable for every price. Something like the charging stats of Tesla app when charging at home!
